docs/contributing/code-of-conduct.md

```diff
@@ -1,76 +1,62 @@
+- अन्य आचरण जिन्हें प्रोफेशनल सेटिंग में यथोचित रूप से अनुचित माना जा सकता है
 
-## Our Responsibilities
+## हमारी जिम्मेदारियां
 
-Project maintainers are responsible for clarifying the standards of acceptable
-behavior and are expected to take appropriate and fair corrective action in
-response to any instances of unacceptable behavior.
+प्रोजेक्ट मेंटेनर की ज़िम्मेदारी है की वो स्वीकार करने लायक व्यवहार का स्टैण्डर्ड स्पस्ट करें
+और उनसे किसी भी तरह के अस्वीकारिये व्यवहार के जवाब में उचित और सही कार्रवाई करने की उम्मीद है।
 
-Project maintainers have the right and responsibility to remove, edit, or reject
-comments, commits, code, wiki edits, issues, and other contributions that are
-not aligned with this code of conduct, or to ban temporarily or permanently any
-contributor for other behaviors that they deem inappropriate, threatening,
-offensive, or harmful.
+प्रोजेक्ट मेंटेनर के पास किसी भी कमेंट, कमिट, कोड, wiki बदलाव, issues, और कोई भी योगदान जो हमारी आचार संहिता की अनुसार नहीं है  उसे हटाने, बदलने या अस्वीकार करने या किसी भी योगदानकर्ता के अनुचित, धमकी, आक्रामक, या हानिकारक व्यवहार को देख कर अस्थायी या स्थायी रूप से प्रतिबंध का हक और जिम्मेदारी है।
 
-## Scope
+## क्षेत्र
 
-This Code of Conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community. Examples of
-representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
-representative at an online or offline event. Representation of a project may be
-further defined and clarified by project maintainers.
+यह आचार संहिता प्रोजेक्ट के भीतर और सार्वजनिक स्थानों पर लागू होती है
+जब भी कोई व्यक्ति प्रोजेक्ट या उसके समुदाय का प्रतिनिधित्व करता है। एक आधिकारिक प्रोजेक्ट ई-मेल का उपयोग करना, 
+एक आधिकारिक सोशल मीडिया अकाउंट के माध्यम से पोस्ट करना, 
+या एक ऑनलाइन या ऑफलाइन घटना में नियुक्त प्रतिनिधि के रूप में कार्य करना ये सब एक प्रोजेक्ट या समुदाय का प्रतिनिधित्व 
+करने के कुछ उद्धरण हैं। किसी प्रोजेक्ट या प्रतिनिधित्व के बारे में आगे परिभाषित और स्पष्ट रूप से प्रोजेक्ट मेंटेनर्स द्वारा बताया जाएगा।
 
-## Enforcement
+## प्रवर्तन
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at
-[conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com). All complaints will be
-reviewed and investigated and will result in a response that is deemed necessary
-and appropriate to the circumstances. The project team is obligated to maintain
-confidentiality with regard to the reporter of an incident. Further details of
-specific enforcement policies may be posted separately.
+अपमानजनक, उत्पीड़न, या अन्यथा अस्वीकार्य व्यवहार की सूचना प्रोजेक्ट टीम को यहाँ 
+[conduct@gatsbyjs.com](mailto:conduct@gatsbyjs.com) संपर्क करके दी जा सकती है। सभी शिकायतें ड़याँ से देखा और 
+उसकी जांच की जाएगी और उसका परिणाम आवश्यक और परिस्थितियों के लिए उपयुक्त एक प्रतिक्रिया के रूप में होगा। 
```

Comment:
शिकायतें ड़याँ से देखा और उसकी => शिकायतों की समीक्षा और जांच

Reply:
I have updated it to this 
`शिकायतें ध्यान से देखा और उसकी`

I feel this is a bit complex `शिकायतों की समीक्षा और जांच`

Any thoughts?